mmrazor/models/architectures/connectors/mgd_connector.py

```diff
@@ -0,0 +1,63 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from typing import Dict, Optional
+
+import torch
+import torch.nn as nn
+
+from mmrazor.registry import MODELS
+from .base_connector import BaseConnector
+
+
+@MODELS.register_module()
+class MGDConncetor(BaseConnector):
```

Comment:
Maybe `MGDConncetor` should be `MGDConnector`.
Add the test case for the `MGDConnector` in [test_connectors](https://github.com/open-mmlab/mmrazor/blob/dev-1.x/tests/test_models/test_architectures/test_connectors/test_connectors.py)